packages/host/app/styles/app.css

```diff
@@ -674,6 +674,39 @@ nav>.directory-level {
   outline: none;
 }
 
+.operator-mode__add-card {
+  height: calc(100% - var(--search-sheet-closed-height));
+  width: 100%;
+  max-width: 50rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.operator-mode__add-card__title {
+  color: var(--boxel-light);
+  font: var(--boxel-font-lg);
+}
+
+.operator-mode__add-card__button {
+  height: 350px;
+  width: 200px;
+  vertical-align: middle;
+  
+  background: var(--boxel-teal);
+  border: none;
+  border-radius: var(--boxel-border-radius);
+}
+
+.operator-mode__add-card__button:hover {
+  background: var(--boxel-dark);
```

Comment:
<img width="345" alt="image" src="https://github.com/cardstack/boxel/assets/273660/0d0bdb93-f93b-46ad-b825-4378a02b4c5d">
<img width="304" alt="image" src="https://github.com/cardstack/boxel/assets/273660/0edefb3f-848b-4383-adc2-6bbf384f86ac">

It looks a bit jarring when the background changes to black. I suggest using a slightly lighter or darker primary color on hover

Reply:
From our design system docs:

<img width="280" alt="image" src="https://github.com/cardstack/boxel/assets/273660/8e504743-1792-49e5-8873-8eb211d02279">

Reply:
I asked about the hover state for this: https://app.zeplin.io/project/645c0e48370ce64121184d54/screen/645c0e7e6b52b10e5c8a477b

Reply:
Updated https://github.com/cardstack/boxel/pull/345/commits/305b950c0e0bf65fbc9b673b94308027770d22a2